x-pack/plugins/security_solution/public/common/components/threat_match/helpers.test.tsx

```diff
@@ -252,10 +264,11 @@ describe('Helpers', () => {
       const expected: { updatedEntry: Entry; index: number } = {
```

Comment:
Nit: would it be better to update this to `Entry & { id: string }` rather than the `as Entry` below?

Reply:
Yes, thank you